src/integration-tests/test_reconfigure_domains.py

```diff
@@ -33,8 +33,8 @@
 
 INITIAL_MSG_QUOTA = 10
 
-URI_PRIORITY_1 = f"bmq://{tc.DOMAIN_PRIORITY}/abcd-queue"
-URI_PRIORITY_2 = f"bmq://{tc.DOMAIN_PRIORITY}/qrst-queue"
+URI_PRIORITY_1 = f"bmq://{tc.DOMAIN_PRIORITY_SC}/abcd-queue"
+URI_PRIORITY_2 = f"bmq://{tc.DOMAIN_PRIORITY_SC}/qrst-queue"
```

Comment:
Don't we want to test non-SC domain here anymore?
I am thinking if we can parametrize this with tweaks.
I am also ok if we just switch to SC here without extra work, or we can add an issue to support non-SC if it's worth it

Reply:
The problem of parameterize testing non-SC and SC domains exist for many integration test files. Will create an issue to support all of them

Reply:
Is there a significant difference in the broker logics so we are better to support both? If we don't expect differences in the logics, might just leave SC-one

Reply:
There is minimal difference in logics. I would advocate always testing SC mode, as non-SC is considered legacy.

Reply:
The difference is SC domains are more asynchronous as they need to wait for quorum of receipts from replicas. I will create an issue.

Reply:
https://github.com/bloomberg/blazingmq/issues/543